action/find.go

```diff
@@ -18,11 +20,33 @@ func (s *Action) Find(c *cli.Context) error {
 		return err
 	}
 	needle := strings.ToLower(c.Args().First())
+	choices := make([]string, 0, 10)
 	for _, value := range l {
 		if strings.Contains(strings.ToLower(value), needle) {
+			choices = append(choices, value)
+		}
+	}
+
+	if len(choices) == 1 {
+		fmt.Println(color.YellowString("Found exact match in '%s'", choices[0]))
```

Comment:
I think it would be better to have this not print a yellow string too.
Everything returned by this is yellow and the different messages are hard to distinguish.
What about making this line green?